vowpalwabbit/gd.cc

```diff
@@ -498,6 +503,8 @@ float get_pred_per_update(gd& g, example& ec)
       }
     nd.pred_per_update *= g.update_multiplier;
   }
+  if (nanpattern(nd.pred_per_update) || infpattern(nd.pred_per_update))
```

Comment:
Another strange test.  Is this for debugging purposes?  This seems like the kind of thing which covers up a bug that should be addressed more directly.